### DISCUSSION TITLE
how to edit with Figma

### DISCUSSION THREAD

**Discussion by onurerisgen** [ColorlibHQ/AdminLTE#4732] [Q&A] [answered]
Hi,
For our project we want to edit AdminLTE (or a paid version) with Figma to create a more dedicate and specialized ui. we want also add some new widgets, components, etc.
For this purpose we need a Figma output file of AdminLTE. How can i get it?

I found only https://icons.getbootstrap.com/ for Figma outputs of Bootstrap icons.

**Comment by REJack** [comment, marked as answer]
AdminLTE doesn't have a Figma UI Kit, I even not know if there is a easy way to create it.

**Comment by idhamhafidz** [comment]
Simply use a web to figma Plugins. You can get it all done in minutes.
